src/index.ts

```diff
@@ -165,7 +166,11 @@ export async function main(
     const gcovLogs = await generateGcovCoverageFiles(projectRoot, gcovInclude, gcovIgnore, gcovArgs)
     UploadLogger.verbose(`${gcovLogs}`)
   }
-  
+
+  UploadLogger.verbose('Running coverage xml...')
+  const coveragePyLogs = await generateCoveragePyFile()
+  UploadLogger.verbose(`${coveragePyLogs}`)
```

Comment:
Should this be behind a feature flag of some kind?

Reply:
No, I don't think so, I think we should be doing this everytime